### PR TITLE
Script to install/uninstall packages

### PR DESCRIPTION
**Pull request by AbdullahM0hamed**
This script calls the package installer via Intents and requires the following permissions to be in termux-api:

`android.permission.REQUEST_INSTALL_PACKAGES`
`android.permission.REQUEST_DELETE_PACKAGES`

### CODE REVIEW

**Comment by AbdullahM0hamed** [comment]
@xalexalex, it is a typo, yes.

I couldn't figure out how to change it though, unfortunately.

**Comment by AbdullahM0hamed** [comment]
I'm not sure what I was doing wrong before, but I've managed to change the name. I've also rearranged the logic as requested.

File: scripts/termux-apk [hunk 1]
```diff
@@ -0,0 +1,50 @@
+#!/data/data/com.termux/files/usr/bin/bash
+
+install()
+{
+    apk="$(realpath $@)"
```
Comment (nikitabobko) [inline scripts/termux-apk]: This function is supposed to be called with only one argument. But, `$@` gives false sense that this function can be called with several arguments. I'd better change it to `$1`. Or, which is certainly preferable I'd add support for several args passed to `-i` and `-u` and in this function iterate over `$@`

File: scripts/termux-apk [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    then
+	echo "$apk does not exist"
+	exit 1
+    fi
+    am start -a android.intent.action.INSTALL_PACKAGE -d "file://$apk"
+}
+
+uninstall()
+{
+    am start -a android.intent.action.UNINSTALL_PACKAGE -d package:"$@"
```
Comment (nikitabobko) [inline scripts/termux-apk]: The same about `$@` here

File: scripts/termux-apk [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+install()
+{
+    apk="$(realpath $@)"
+    if [ ! -f "$apk" ]
+    then
+	echo "$apk does not exist"
+	exit 1
+    fi
+    am start -a android.intent.action.INSTALL_PACKAGE -d "file://$apk"
```
Comment (nikitabobko) [inline scripts/termux-apk]: I'd propagate exit code of `am` command with `return $?` and put `set -e` at the start of the script. The same about `uninstall` function

File: scripts/termux-apk [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    echo "Usage: termux-apk [command] [apk/package]"
+    echo -e "\t-i: Install an apk. Following argument must be the path to an apk"
+    echo -e "\t-u: Uninstall a package. Following argument must be a package name."
+    exit 1
+}
+
+
+if [ $# != 2 ]
+then
+       show_usage
```
Comment (nikitabobko) [inline scripts/termux-apk]: What's wrong with codestyle? it's 8 spaces. But at the same time you use 4 spaces in other places in this file. Also tabs mixed with spaces across the file

**Comment by AbdullahM0hamed** [comment]
@nikitabobko, I've addressed your concerns. Regarding that last concern I edited this using emacs (in termux) and I haven't changed anything from the default settings (I went ahead an manually backspaced it through).

**Comment by Movado04** [comment]
How can I use my termux

**Pull request closed without merging**